src/interviews/utils/interviewHandlers.ts

```diff
@@ -0,0 +1,42 @@
+import { promises as fs } from 'fs';
+import matter from 'gray-matter';
+
+import path from 'path';
+
+type InterviewMetadata = {
+  meta_title: string;
+  meta_description: string;
+  project: string;
+  projectSlug: string;
+  variant: string;
+  location: string;
+  duration: string;
```

Comment:
Lurer på om duration bør være en `durationFrom` og `durationTill`. Og i det tilfellet `durationTill` ikke er satt så kommer pila `->` (for å presisere at det er pågående)?